icons.cpp

```diff
@@ -0,0 +1,60 @@
+/****************************************************************************
```

Comment:
why this file is in this PR as new?

Reply:
It was removed before, must have been recovered during merge. I will remove it once again, it is not needed any more.